### PR TITLE
feat(server): error message when trying to import a schema with exist…

### PR DESCRIPTION
**Pull request by shaikfareed46**
Close: #6716

## PR Details

This PR will give clear error message when trying to import a schema with an entity that already exists in the service

### Summary
📝🚀🛠️

Improve error message for entity name conflict in `EntityService`. This is part of a feature to import entities from a schema file.

> _`EntityService`_
> _Checks for name collisions_
> _Error message helps_

### Walkthrough
* Improve error message for entity name conflict ([link](https://github.com/amplication/amplication/pull/6717/files?diff=unified&w=0#diff-9a1aca769e1c8ddba4436f7b850d36912e8fa38e946d5c40d68631be40167e59L556-R556))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by overbit** [comment]
@shaikfareed46 thanks for your first contribution. 
Can you please follow the CONTRIBUTING.md documentation and fill the necessary details in this PR?

**Comment by shaikfareed46** [comment]
> @shaikfareed46 thanks for your first contribution. Can you please follow the CONTRIBUTING.md documentation and fill the necessary details in this PR?

The PR is updated. Please verify and let me know if there any changes required to do.

File: packages/amplication-server/src/core/entity/entity.service.ts [hunk 1]
```diff
@@ -553,7 +553,7 @@ export class EntityService {
     if (existingEntities.length > 0) {
       existingEntities.forEach((entity) => {
         void actionContext.onEmitUserActionLog(
-          `Entity "${entity.name}" already exists`,
+          `Entity "${entity.name}" already exists, exists in the service. To proceed with the import, please rename or remove the entity in your schema file or remove the conflicting entity from the service.`,
```
Comment (overbit) [inline packages/amplication-server/src/core/entity/entity.service.ts:556]: Be careful with the requirement:

```suggestion
          `Entity "${entity.name}" already exists in the service. To proceed with the import, please rename or remove the entity in your schema file or remove the conflicting entity from the service`,
```

**Review by overbit — CHANGES_REQUESTED** [review]
@shaikfareed46 small comment

File: packages/amplication-server/src/core/entity/entity.service.ts (hunk 1 above)
Comment (shaikfareed46) [reply packages/amplication-server/src/core/entity/entity.service.ts]: @overbit  Thank you for noticing and pointing out,  I have made the changes.

**Pull request merged**